projects/components/src/titled-content/titled-content.component.ts

```diff
@@ -90,7 +90,8 @@ export const enum TitlePosition {
 // Regular title (black) and GrayedOut new style (gray)
 export const enum TitledContentTitleStyle {
   Regular = 'regular',
-  GrayedOut = 'grayed-out'
+  GrayedOut = 'grayed-out',
+  TitleCasedGray = 'title-case-gray'
```

Comment:
nit: should this be `title-cased-gray` or it is intended?

Reply:
It was intended, but I hadn't noticed the discrepancy earlier. Will fix in a follow up.